spec/weather-bot/services/dark_sky_service_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
```

Comment:
I test the structure of the incoming JSON object from the DarkSky API in each of these tests. I don't test the actual values b/c the weather is constantly changing which would break the tests. What is your opinion on this and should I be testing anything in addition to the structure?